lighthouse-core/config/config.js

```diff
@@ -145,6 +145,27 @@ function validatePasses(passes, audits, rootPath) {
   });
 }
 
+function validateCategories(categories, audits, auditResults = []) {
```

Comment:
it shouldn't be possible for `auditResults` to ever be undefined. At worst it'll be [`null`](https://github.com/GoogleChrome/lighthouse/blob/9349c894a6a9d4f9b1f0a74ce4555433591d98aa/lighthouse-core/config/config.js#L288), so it won't ever get the default value